taxonomies/labels.txt

```diff
@@ -18247,6 +18248,9 @@ fi:liiallisella kulutuksella saattaa olla laksatiivisia vaikutuksia, liiallisell
 fr:Une consommation excessive peut avoir des effets laxatifs
 it:Un consumo eccessivo puù avere effetti lassativi
 
+en:In this factory sesame seeds, sunflower seeds and nuts are used
```

Comment:
Sentences that indicate possible traces should not be added to labels, instead we try to parse them.

I filed a bug for this one: https://github.com/openfoodfacts/openfoodfacts-server/issues/3592

Reply:
ok